suzieq/poller/worker/nodes/node.py

```diff
@@ -1266,7 +1266,9 @@ async def _init_ssh(self, init_boot_time=True,
                         self.logger.info(
                             'Privilege escalation succeeded for '
                             f'{self.hostname}')
-            except Exception:
+            except Exception as e:
+                self.logger.error('Unable to create persistent SSH session'
+                                  f' for {self.hostname} due to {str(e)}')
```

Comment:
Shouldn't we also set `self._current_exception`?

Reply:
Yes, will fix